docs/reference/JpegPresets.rst

```diff
@@ -0,0 +1,11 @@
+.. py:currentmodule:: PIL.JpegPresets
+
+:py:mod:`JpegPresets` Module
+============================
+
+.. automodule:: PIL.JpegPresets
+
+    .. data:: presets
+        :type: dict
+
+        A dictionary of all supported presets.
```

Comment:
👍 

We can consider improving the formatting of this in a later PR.

https://pillow--4697.org.readthedocs.build/en/4697/reference/JpegPresets.html

https://github.com/python-pillow/Pillow/blob/master/src/PIL/JpegPresets.py